ansible/ome-analysis/ome-jupyter.yml

```diff
@@ -0,0 +1,104 @@
+---
+# This is an almost self-contained playbook for setting up a single
+# Jupyterhub server, with cadvisor running on port 9280
+# The only external variables are for GitHub OAuth.
+# In future if this is used to manage a cluster of Jupyterhub servers
+# some of the hardcode variables (e.g. volume sizes) should be moved into
+# group_vars
+
+- hosts: ome-jupyter
+
+  roles:
+
+  # Do we need this?
+  #- role: server-swap
```

Comment:
IIRC there was a problem with some centos 6 machines that unexpectedly used swap. I don't know if it's needed on centos 7. @kennethgillen ?